packages/esm-patient-forms-app/src/clinical-form-action-menu.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Document20 from '@carbon/icons-react/es/document/20';
+import styles from './clinical-form-action-menu.scss';
+import { useTranslation } from 'react-i18next';
+import { useLayoutType } from '@openmrs/esm-framework';
+import { Button } from 'carbon-components-react';
+
+const ClinicalFormActionMenu: React.FC = () => {
+  const { t } = useTranslation();
+  const layout = useLayoutType();
+
+  if (layout === 'tablet')
+    return (
+      <Button kind="ghost" className={styles.container} onClick={() => {}}>
```

Comment:
What's with the empty `onClick`?

Reply:
The clinical form workspace hasn't been implemented yet as seen here in the [designs](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/61f040dbd4e9d1a258f4e157) should this Action button be removed?